src/python/ensembl/production/factories/metadata/genomeFactory.py

```diff
@@ -20,451 +20,524 @@
 import re
 import graphene
 from graphene_sqlalchemy import SQLAlchemyObjectType
-
-from sqlalchemy import create_engine
-from sqlalchemy.orm import sessionmaker
+from graphene_sqlalchemy.converter import convert_sqlalchemy_type
 from sqlalchemy import select
-from sqlalchemy import update
-
-from ensembl.database import DBConnection
+from sqlalchemy import JSON
 from ensembl.production.metadata.api.models.genome import Genome, GenomeDataset
 from ensembl.production.metadata.api.models.organism import Organism, OrganismGroup, OrganismGroupMember
 from ensembl.production.metadata.api.models.assembly import Assembly
-from ensembl.production.metadata.api.models.dataset import DatasetType, Dataset, DatasetSource
-
-logging.basicConfig(level=logging.INFO, format='%(message)s')
-logger = logging.getLogger(__name__)
+from ensembl.production.metadata.api.models.dataset import DatasetType, Dataset, DatasetSource, DatasetStatus
+from ensembl.production.metadata.api.hive.dataset_factory import DatasetFactory
```

Comment:
I suspect this dependency path is not available any more. following latest updates in api code structure.

Reply:
yes, changes are needed for the genome factory and hive module, I will  update accordingly

Reply:
Wait for the next tag which should arrive today the. Thanks!